packages/cerebral/src/Controller.js

```diff
@@ -71,21 +79,16 @@ class Controller extends EventEmitter {
       throw error
     })
 
-    if (this.devtools) {
-      this.devtools.init(this)
-    } else if (
-      isDeveloping() &&
-      typeof navigator !== 'undefined' &&
-      /Chrome/.test(navigator.userAgent)
-    ) {
-      console.warn('You are not using the Cerebral devtools. It is highly recommended to use it in combination with the debugger: https://cerebral.github.io/cerebral-website/install/02_debugger.html')
-    }
-
-    if (
-      isDeveloping() &&
-      this.options.strictRender
-    ) {
-      console.info('We are just notifying you that STRICT RENDER is on')
+    if (process.env.NODE_ENV !== 'production') {
+      if (this.devtools) {
```

Comment:
I just tested combining these expression but than uglify wont remove the branch.